clipper_admin/clipper_admin/kubernetes/kubernetes_container_manager.py

```diff
@@ -632,6 +632,25 @@ def stop_models(self, models):
                             val=create_model_container_label(m, v),
                             cluster_label=CLIPPER_DOCKER_LABEL,
                             cluster_name=self.cluster_name))
+
+                    self._k8s_beta.delete_collection_namespaced_replica_set(
+                        namespace=self.k8s_namespace,
+                        label_selector=
```

Comment:
It seems there are redundant codes. Would like to define `label_selector` as a variable?

Reply:
Sure, I copy-pasted the parts so it's not pretty, I will fix it now :)